src/management/apps.ts

```diff
@@ -224,3 +234,64 @@ function getListAppsResourceString(projectId: string, platform: AppPlatform): st
 
   return `/v1beta1/projects/${projectId}${resourceSuffix}`;
 }
+
+/**
+ * Gets the configuration artifact associated with the specified a Firebase app
+ * @return a promise that resolves to configuration file content of the requested app
+ */
+export async function getAppConfig(
+  appId: string,
+  platform: AppPlatform
+): Promise<AppConfigurationData> {
+  try {
+    const response = await api.request("GET", getAppConfigResourceString(appId, platform), {
+      auth: true,
+      origin: api.firebaseApiOrigin,
+      timeout: TIMEOUT_MILLIS,
+    });
+    return parseConfigFromResponse(response.body, platform);
+  } catch (err) {
+    logger.debug(err.message);
+    throw new FirebaseError(
+      `Failed to get ${platform} app configuration. See firebase-debug.log for more info.`,
+      {
+        exit: 2,
+        original: err,
+      }
+    );
+  }
+}
+
+function getAppConfigResourceString(appId: string, platform: AppPlatform): string {
+  let platformResource;
+  switch (platform) {
+    case AppPlatform.IOS:
+      platformResource = "iosApps";
+      break;
+    case AppPlatform.ANDROID:
+      platformResource = "androidApps";
+      break;
+    case AppPlatform.WEB:
+      platformResource = "webApps";
+      break;
+    default:
+      throw new FirebaseError("Unexpected app platform");
+  }
+
+  return `/v1beta1/projects/-/${platformResource}/${appId}/config`;
+}
+
+function parseConfigFromResponse(responseBody: any, platform: AppPlatform): AppConfigurationData {
+  if (platform === AppPlatform.WEB) {
+    return {
+      fileName: WEB_CONFIG_FILE_NAME,
+      fileContents: inspect(responseBody, { compact: false }),
```

Comment:
@bkendall what's the best practice for importing methods like this? IMO it's more readable inline if `util` itself was imported, and `util.inspect(...)` was used inline here.

Reply:
I agree that `util.inspect(..)` seems to be more readable. Changing the code now, can revert if @bkendall thinks it's better the other way

Reply:
@bkendall, how do you think?